powerlibs/django/restless/contrib/endpoints/base.py

```diff
@@ -71,6 +71,40 @@ def get_query_set(self, request, *args, **kwargs):
 
 
 class SoftDeletableDetailEndpointMixin:
+    def serialize(self, data):
+        data = super().serialize(data)
+        if 'deleted' in data:
+            del data['deleted']
+
+        return data
+
+    def put(self, request, *args, **kwargs):
```

Comment:
1- `created_by` and other fields have **nothing** to do with this mixin.
2- Reimplementing almost entirely the `put` method. It seems too agressive.
3- Two calls to the "parent" method (with `super`) on two different returns. It's a bad smell.
4- Why would you fill `result` (line 83) if it is not even used to check for invalid fields?

So, please, take a deep breath, think about what you want to do, and try again.